roles/ovirt-ova-export-pre-pack/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I think I've accidentally forgot to sent the most important comment.
I guess you can about performance here, so I think the better option for you would be to do those checks in the python script and remove both `ovirt-ova-export-pre-pack` `ovirt-ova-export-post-pack`. I think you can do this in python script. It will be more performant, as all of those tasks must package the python script(ansible module) and tranfer it via SSH to host, so you can safe 4 file transfer to host, if you will do those checks in the script.

Reply:
@machacekondra , I can put move the validations to the python script.
But let's say that we or the storage guys will next add the ability to export the VM to an OVA that is generated inside a pre-made disk image. In that case, we don't need to execute these validations and we don't need to rename the file. So if I move the content of both pre-pack and post-pack inside the python script, we would need different ansible role with a different python script that shares most of the code of this one. Would it be acceptable to merge such a thing later on? I just thought that ansible roles were meant to be reusable in different playbooks and so this design made sense to me, but I see your point regarding the performance.

Reply:
Ah, OK, I understand. I just wanted to point out, that it could be more efficient. But I understand why you did it like this, definitely better when the script could be reused.

Reply:
@machacekondra , maybe you can help me with an issue I'm having - I posted a patch that adds the playbook and the roles to the engine's repository (https://gerrit.ovirt.org/#/c/80616/, shall we continue the review on gerrit btw?) and then the execution of the playbook fails with an error which says that the log file doesn't exist (the same code worked as part of this PR), so I commented out the log part and then it worked again (https://gerrit.ovirt.org/#/c/82526/40/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/CreateOvaCommand.java, line 122-125), did you encounter such a issue while moving other playbooks?

Reply:
@machacekondra actually there is a difference between the code on gerrit and the one here, I renamed the log directory (pack-ova -> create_ova) but I verified that this directory exists with the right permissions...

Reply:
@ahadas Hmm, I didn't have such issue. If you commented out the log part, could you then see the log file in `var/log/ansible/ansible-{date}-{hostanme}.log` ? If yes, maybe there is some strange chars in the path.. can you share the log of ovirt-engine ?

And yes, if you can move the development to gerrit it would be better, as we already moved all the playbooks there, so if possible you can close it here. Thanks!

Reply:
@machacekondra , I found the issue - the log's directory was named create_ova and in the code I wrote create-ova... thanks :) so closing this PR, and we'll continue on gerrit.